db/methods/identity_verification_method/get_by_identity.js

```diff
@@ -0,0 +1,10 @@
+const { getDocument } = require('../../dynamo');
+const IdentityVerificationMethod = require('../../schemas/identity_verification_method');
+
+function getIdentityVerificationMethodByIdentity(identityKey) {
```

Comment:
Right, the intention was to use the `identity` suffix to denote a method that fetches by the hash key + range key (if applicable). I think there's not a conflict here since the `getRecoveryMethodByIdentity` is for a different model.